tests/e2e/no_quorum_notify_daemon_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"testing"
+
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
+)
+
+func TestInitDaemonNotifyWithoutQuorum(t *testing.T) {
+	// Initialize a cluster with 3 members
+	epc, err := e2e.InitEtcdProcessCluster(t, e2e.NewConfigAutoTLS())
+	if err != nil {
+		t.Fatalf("Failed to initilize the etcd cluster: %v", err)
+	}
+
+	// Remove two members, so that only one etcd will get started
+	epc.Procs = epc.Procs[:1]
+
+	// Start the etcd cluster with only one member
+	if err := epc.Start(); err != nil {
+		t.Fatalf("Failed to start the etcd cluster: %v", err)
+	}
+
+	// Expect log message indicating time out waiting for quorum hit
+	e2e.ExpectLog(t, epc.Procs[0], "startEtcd: timed out waiting for the ready notification")
+	// Expect log message indicating systemd notify message has been sent
+	e2e.ExpectLog(t, epc.Procs[0], "notifying init daemon")
```

Comment:
It this sufficient signal for us to confirm that `ExperimentalWaitClusterReadyTimeout` works? I think the idea for the flag was to allow serializable requests without quorum. Do we have a test for that? If not, could you just add serializable request to this test?

Reply:
> I think the idea for the flag was to allow serializable requests without quorum. Do we have a test for that?

Yes, we have added test for that long ago (last year?). This PR is just to enhance the https://github.com/etcd-io/etcd/pull/13525 to address https://github.com/etcd-io/etcd/issues/13785#issuecomment-1226776763

Reply:
Awesome, can we have the both tests in one file? They test the same feature so it makes sense they are close to each other.

Reply:
I thought about it, but it isn't feasible. I recall that previous tests run for both e2e and integration. But this test only run for e2e.

Reply:
:( ok

Reply:
[kv_test.go#L202](https://github.com/etcd-io/etcd/blob/f00578c3cdb848fd8c4449899aebfe6bbc5dd2c6/tests/common/kv_test.go#L202)

The existing test for reference.  It got moved in to common when it was just e2e in the original MR